### ISSUE TITLE
Active element style not present when using NVDA

### ISSUE THREAD

**Issue by smoralesd** [microsoft/accessibility-insights-web#1569]
## Describe the bug

While using NVDA and opening the "more actions" kebab menu on the failure instance card, the contextual menu automatically selects the first element **File issue** but there is no visual indication.

**To Reproduce**

1. Go to [Accessible University before](http://www.washington.edu/accesscomputing/AU/before.html)
1. Open **Fast pass**, navigate to **Automated checks** and wait for the results to show up.
1. Navigate to the first rule group **bypass**
1. Expand the **bypass** group
1. Navigate to the first card (path **html**) and then inside the card until you reach the "more actions" button.
1. Activate the button
1. Observer: NVDA will announce **File Issue 1 of 2** which is correct but there is not visual indication that **File issue** menu item has the focus or is the active element.

## CodePen repro example

This CodePen was created using "Export to CodePen" feature from Office UI Fabric website

https://codepen.io/smoralesd/pen/XWWaemX

## Expected behavior

Using AT should not affect the visual stylings on the contextual menu.

## Screenshots

**No AT**
![01 - no AT](https://user-images.githubusercontent.com/2837582/67706599-4b856000-f976-11e9-8580-8670f21030d2.gif)

**NVDA**
![02 - NVDA](https://user-images.githubusercontent.com/2837582/67706603-4e805080-f976-11e9-98aa-d93a8eeff607.gif)

## Extension (please complete the following information)

- OS: Windows 10.0.18363 Build 18363
- Version Insider 2.10.3
- Browser Microsoft Edge version 79.0.308.1
- Target Page [Accessible University before](http://www.washington.edu/accesscomputing/AU/before.html)

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

No

## Additional context

This is not happening on prod (2.10.0) as there is no **More actions** buttons yet.

**Comment by smoralesd** [comment]
@iamrafan I was working to file this issue against office ui fabric react repo but they have this in their ***Accessibility Issue** template:
> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

Can you help us check if we should file this agains **NVDA** and **JAWS** (it repro on both).

In the case of **Narrator**, the focus indicator style is being override so "it looks" like is working but it's now the normal style.

**Comment by ferBonnin** [comment]
@iamrafan why is this marked as blocked?

**Comment by ferBonnin** [comment]
Removing blocked tag since there is no reason for this to be blocked. 
@iamrafan investigating this. Per conversation with him, seems to be a known Jaws issue and possibly a known NVDA issue. Please add a link to the bugs.

**Comment by iamrafan** [comment]
@ferBonnin this was the issue I mentioned was a known JAWS issue, as I was able to see the focus indicator if I manually switched modes and navigated to the ```menuitems```. But after investigating https://github.com/microsoft/accessibility-insights-web/issues/1568, this looks a similar office fabric issue

@smoralesd Even here the ```ms-Fabric--isFocusVisible``` class is removed when activating the ''more options'' button

Interestingly the class is not removed and the focus is visible if I manually switch to forms mode and activate the "more options" button

**Comment by ferBonnin** [comment]
@iamrafan lets validate this in latest version of OF and check if this is fixed or we need to file an issue with Office Fabric.

**Comment by ferBonnin** [comment]
@iamrafan can you check if this is fixed in Playground?

**Comment by iamrafan** [comment]
@ferBonnin I validated in Playground version 2020.1.8.2334. This issue still exists

**Comment by iamrafan** [comment]
@ferBonnin  Filed https://github.com/OfficeDev/office-ui-fabric-react/issues/11793

**Comment by iamrafan** [comment]
@smoralesd, can you please confirm if a similar fix like you did for #1568 , will work for this issue too?

**Comment by iamrafan** [comment]
> @ferBonnin Filed [microsoft/fluentui#11793](https://github.com/microsoft/fluentui/issues/11793) 

The external issue fix also fixed this issue. Please find the validation details from below.

```AI Web Version 2.27.0```
```NVDA Version 2021.1```
```Google Chrome Version 92.0.4515.131```

![NVDA more options](https://user-images.githubusercontent.com/22068434/129412019-dd5ec3fe-cf0a-4c17-aa58-300278fc933c.gif)
